code/game/machinery/spaceheater.dm

```diff
@@ -140,8 +140,10 @@
 	if(mode == HEATER_MODE_COOL)
 		delta_temperature *= -1
 	if(delta_temperature)
-		enviroment.temperature += delta_temperature
-		air_update_turf(FALSE, FALSE)
+		for (var/turf/open/turf in ((local_turf.atmos_adjacent_turfs || list()) + local_turf))
```

Comment:
Just to make sure, this checks if it could reasonably interact with tiles near it, correct? The example I was thinking of was putting a space heater up against directional windows, it won't heat the turfs on the other side of the windows when it shouldn't be right?